src/syscheckd/src/whodata/win_whodata.c

```diff
@@ -1089,7 +1088,8 @@ int policy_check() {
 
 error:
     err_code = GetLastError();
-
+    char err_msg[OS_SIZE_1024];
+    err_msg[OS_SIZE_1024 - 1] = '\0';
```

Comment:
I would recommend changing this in favor of:
```suggestion
    char err_msg[OS_SIZE_1024];
    memset(err_msg, 0, OS_SIZE_1024); // Set all elements to zero
```
I suggest changing it as well in other occurrences.